neuralcompression/data/vimeo90k.py

```diff
@@ -28,22 +28,28 @@ class Vimeo90kSeptuplet(torch.utils.data.Dataset):
     Args:
         root: Path to the Vimeo-90k root directory (i.e. the
             directory containing the dataset's README).
-        as_video: Determines whether the dataset should return individual images
-            (as_video=False) or multiple consecutive frames (as_video=True) at
-            a time.
-        frames_per_group: The number of frames to include from each septuplet.
-            Specifically, the first frames_per_group frames from each septuplet
-            are included in the dataset. Must be between 1 and 7.
+        as_video: Determines whether the dataset should return
+            individual images (``as_video=False``) or
+            multiple consecutive frames  (``as_video=True``)
+            at a time.
+        frames_per_group: The number of frames to include from
+            each septuplet. Specifically, the first ``frames_per_group``
+            frames from each septuplet are included in
+            the dataset. Must be between 1 and 7.
         split: Specifies which dataset parition should be used. Valid values
-            are "train" or "test". Exactly one of ``split`` or
+            are ``"train"`` or ``"test"``. Exactly one of ``split`` or
             'folder_list' must be specified.
         folder_list: A list of paths to septuplets to include in the dataset
             split. Each septuplet path must be a directory containing the files
-            im1.png, ..., im7.png. Exactly one of ``split`` or
+            ``im1.png``, ..., ``im7.png``. Exactly one of ``split`` or
             ``folder_list`` must be specified.
         pil_transform: Callable object for applying transforms to
             the PIL images prior to image concatenation. If using, be sure to
             have the final operation convert the PIL image to a tensor.
+            Following ``torchvision``'s dataset conventions, the default
+            transform is ``torchvision.transforms.ToTensor`` when
+            in video mode (i.e. when ``as_video=True``), while no
+            default transform is applied in image mode.
```

Comment:
Could you also add a comment on this using `Note:` before `Args`? Example: https://github.com/facebookresearch/NeuralCompression/blob/e1702923c97cc6c00cd8af4db2eb59f587c09e44/neuralcompression/functional/information.py#L24-L26